builder/index.html

```diff
@@ -62,11 +62,14 @@ <h3 class="modal-title">{{actionName}} {{kind}}</h3>
               <h3 class="modal-title">Clone Technique</h3>
             </div>
             <div class="modal-body">
-              <div class="form-group">
+              <div class="form-group" ng-class="{'has-error' : !checkLength(technique.name)}">
                 <label for="cloneName" class="control-label">New Technique name:</label>
                 <div>
                   <input  type="text" id="cloneName" name="name" techniquename class="form-control" ng-model="technique.name" placeholder="Technique Name" required>
                 </div>
+                <span class="text-danger" ng-show="!checkLength(technique.name)">
```

Comment:
And using ng-maxlength, you can use here :  ng-show="formName.inputName.$errors.maxlength"